_posts/2024-04-08-march-rubygems-updates.md

```diff
@@ -0,0 +1,77 @@
+
+**[Make `gem install` respect the umask of the target system](https://github.com/rubygems/rubygems/pull/7518)**: 
+
+- The goal of this change is to address the issue where RubyGems may install files with permissions that are broader than desired, giving write permissions to users other than the current user. This issue arises when the original packaging of files includes these broad permissions, likely due to an unsafe umask set by the gem's author.
+- The solution implemented by [@deivid-rodriguez](https://github.com/deivid-rodriguez) was to adopt a more straightforward approach than the previous attempt (which was reverted due to test failures in `ruby core`) by applying the target system’s umask to regular files (excluding directories) before setting their permissions.
+
+**[Fixed Bundler’s application cache misuse](https://github.com/rubygems/rubygems/pull/7516):**
+
+- This update resolves an issue in how Bundler was using its cache, leading to odd behavior. Users were seeing unusual updates, like Bundler claiming it was updating to versions that didn’t actually exist (for example, "Updating to 3.0.9").
+- The problem was rooted in how Bundler managed cached gems. These gems were mistakenly being considered in situations they shouldn’t have been, which caused not only strange messages but also errors in the lockfile, such as gems appearing under incorrect sources.
+- The solution implemented ensures that cached gems are kept separate from those available online, preventing the confusion that was causing these issues. This approach helps maintain clarity and accuracy in Bundler’s operations.
+
+In March, RubyGems gained [67 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2024-3-01%7D...master@%7B2024-3-31%7D) contributed by 13 authors. There were 934 additions and 194 deletions across 92 files.
+
+## RubyGems.org News
+
+March's updates to RubyGems.org reflect a strong commitment to improving user experience, enhancing security, and modernizing the platform.
+
+The following are highlights of what the team worked on this month:
+
+**[Major PostgreSQL zero downtime upgrade](https://github.com/rubygems/pg-major-update)**:
+
+- This significant update was carried out to ensure that application dependencies remain up-to-date. Notably, this is the second upgrade effort, moving from PostgreSQL version 12 to 13, following the original upgrade to version 12 in response to the end of life (EOL) for PostgreSQL 11 on Amazon RDS.
+- The upgrade process utilized `pgbouncer` and a manually managed blue/green environment to achieve zero downtime. For detailed scripts and an explanation of the procedure, visit the [project’s GitHub page](https://github.com/rubygems/pg-major-update).
+- Additional information and related updates can be found in [#4482](https://github.com/rubygems/rubygems.org/pull/4482) and [#4500](https://github.com/rubygems/rubygems.org/pull/4500).
```

Comment:
Can you mention more detailed blog post on this will be published soon instead of those two links on this line?

Reply:
Sure I will update it now.